src/main/kotlin/com/zopa/ktor/opentracing/OpenTracingServer.kt

```diff
@@ -82,6 +84,20 @@ class OpenTracingServer {
                 }
             }
 
+            pipeline.environment.monitor.subscribe(Routing.RoutingCallFinished) {call ->
```

Comment:
can we do subscribe to `RoutingCallStarted` ? We already have an issue where if an exception is thrown in the router, the second intercept doesn't get called ( which means we don't have the status code tag ).

Reply:
Also, this increases the likelihood that the active span is the server span, but we should check anyways

Reply:
How would you check that it's the right span?

Reply:
you can check that it has this server tag; 
https://github.com/zopaUK/ktor-opentracing/blob/ee109fbc7fe6e599f91e75dafe0c36f5f24ad876/src/main/kotlin/com/zopa/ktor/opentracing/OpenTracingServer.kt#L67

Reply:
I might be missing something, but I don't think you can read tags (or span name for that matter) without knowing the implementation of Span 😐